File: app/models/concerns/artist.rb
class Artist < ActiveRecord::Base
    validates :first_name, :presence => true
    validates :last_name, :presence => true
    validates :description, :presence => true
    
    attr_accessible :first_name, :description, :picture, :last_name
  
    has_many :albums, :dependent => :destroy 
end

File: app/assets/javascripts/models/artist.js
StudioSabab.Artist = DS.Model.extend({
	first_name:	 DS.attr("string"),
	last_name: 	 DS.attr("string"),
	description: DS.attr("string"),
	picture: 	 DS.attr("string"),
	
	albums: DS.hasMany('StudioSabab.Album')
});